exercises/alphanumeric-pattern-generator/alphanumeric_pattern_generator_test.rb

```diff
@@ -0,0 +1,156 @@
+require 'minitest/autorun'
+require_relative 'alphanumeric_pattern_generator'
+
+class PatternGeneratorTest < Minitest::Test
+  def test_it_can_verify_a_single_alphabetic_character
+    pattern = '.'
+    pg = PatternGenerator.new(pattern)
+
+    assert_equal true, pg.verify('F')
+    assert_equal false, pg.verify('0')
```

Comment:
try and only have one assertion per test, if you've got more than one then you're probably testing more than one thing and therefore can give each thing it's own descriptive test name.